### ISSUE TITLE
Unlock Guides logo redirects to Unlock Homepage

### ISSUE THREAD

**Issue by patrickworkman** [unlock-protocol/unlock#10715]
**Describe the bug**
When Creators using Guides on Unlock's website click on the 'Unlock Guides' logo, it goes to Unlock's Homepage. This should redirect back to guides. 

**To Reproduce**
1. Click a Guide
2. Click on the Unlock Guides logo
3. Redirect to Unlock's Homepage

**Expected behavior**
Redirect users in Guides to the Guides homepage

**Comment by julien51** [comment]
Yes! @kalidiagne created a re-usable header so we can solve this so I am assigning this to him.
